manual_lock_dep.md

```diff
@@ -0,0 +1,2 @@
+pipenv lock -r > src/python/psic/requirements.txt && 
+pipenv lock -r -d > src/python/psic/requirements-dev.txt
```

Comment:
Some web services and things that use the traditional setup.py for understanding a package's dependencies and contents need the old requirements.txt format and there's no good way to auto generate these without using a pre-commit hook in .git, so this is just a note on the commands that need to be run in the same directory as the Pipfile in order to convert the Pipfile.lock format to requirements.txt format!